api/common/services/UserService.ts

```diff
@@ -85,8 +85,7 @@ export class UserService {
     const userProfileImageLargeUrl = await this.getProfileImage(request.email, 600, request.oauthPicture);
     const requestWithProfileImg = {
       ...request,
-      profileImgSmallUrl: userProfileImageSmallUrl,
```

Comment:
We need both the `profileImgSmallUrl` and `profileImgLargeUrl` properties here. The `profileImgUrl` can be removed.